frontend/src/javascript/utils/projectUtils.js

```diff
@@ -0,0 +1,5 @@
+import { sendRequest } from "@/javascript/utils/axiosUtils";
+
+export function deleteProject(projectId) {
```

Comment:
Ja bym to zostawił tak jak było wcześniej w project store. Warto być spójnym, że albo całą logikę wynosimy do osobnego pliku albo całość jest w tym storze, bo takie przeniesienie jednej (w dodatku małej i prostej) funkcji zbyt wiele nie daje, a w przyszłości może sprawić, że nam się plik rozjedzie, że część logiki będzie tutaj część gdzie indziej. Chyba, że jest jakiś konkretny powód dlaczego tak zrobiłeś. Jak nie, to weź to przywróc spowrotem do store, reszta jest git po tej poprawce PR do akceptacji.

Reply:
Dodałem tę funkcję dlatego, bo nie zawsze projekt, który usuwamy, jest tym który jest w store. 

Jak odpalimy przeglądanie projektów w LoadProjectModal to wtedy jak naciśniemy przycisk do usuwania projektów to jest ta funkcja wykorzystywana (bo projekt, który usuwamy nie jest w naszym store), jeśli usuwamy projekt za pomocą NavigationDrawer'a to usuwany projekt jest w store i z store wykorzystujemy tę funkcję.

No dla mnie też jest to trochę brzydkie rozdzielenie logiki, ale no nie mam innego pomysłu jak uniknąć redundancji kodu w tym przypadku.

TL;DR - Nie zawsze usuwamy projekt jest w store